external/fv3viz/fv3viz/_plot_helpers.py

```diff
@@ -49,6 +49,7 @@ def _min_max_from_percentiles(x, min_percentile=2, max_percentile=98):
     Returns:
         Tuple of values at min_percentile, max_percentile
     """
+    x = np.array(x).flatten()
```

Comment:
Boolean masking could not do multi dimensional data.  This adjustment let me generate the cmap params straight from DataArrays.